components/builder-jobsrv/src/config.rs

```diff
@@ -90,7 +91,7 @@ impl ConfigFile for Config {
                                                connection_db);
         try!(toml.parse_into("cfg.datastore_connection_retry_ms",
                              &mut cfg.datastore_connection_retry_ms));
-        let mut timeout_seconds = 0;
+        let mut timeout_seconds = 3600;
```

Comment:
It's indeed one hour at the moment - the default is:
datastore_connection_timeout: Duration::from_secs(3600),